content/sensu-go/6.5/api/checks.md

```diff
@@ -43,6 +43,9 @@ HTTP/1.1 200 OK
     "subscriptions": [
       "linux"
     ],
+    "pipelines": [
+      "incident_alerts"
```

Comment:
The `pipelines` field for checks & events is a list of `corev2.ResourceReference` instead of a list of strings. For example:

```json
"pipelines": [
  {"api_version": "core/v2", "type": "Pipeline", "name": "incident_alerts"}
]
```

Reply:
The only compatible resource api & type for pipelines at the moment is `core/v2.Pipeline`.